lib/core/config.js

```diff
@@ -221,9 +233,27 @@ Config.prototype.loadContractsConfigFile = function() {
   });
 
   let configFilePath = this._getFileOrOject(this.configDir, 'contracts', 'contracts');
-
-  const newContractsConfig = this._mergeConfig(configFilePath, configObject, this.env);
-
+  let newContractsConfig = this._mergeConfig(configFilePath, configObject, this.env);
+  if (newContractsConfig.gas.match(unitRegex)) {
+    newContractsConfig.gas = utils.getWeiBalanceFromString(newContractsConfig.gas, web3);
+  }
+  if ('accounts' in newContractsConfig.deployment) {
+    newContractsConfig.deployment.accounts.forEach((account, index) => {
+      if (account.balance.match(unitRegex)) {
+        account.balance = utils.getWeiBalanceFromString(account.balance, web3);
```

Comment:
Since you handle balances here, you can remove it from modules/blockchain_process/dev_funds.js

Reply:
@jrainville thanks for the review! Please correct me if I'm wrong, but I think we should keep the conversion in modules/blockchain_process/dev_funds.js, since that is converting the account balance in the blockchain configuration, while here we are converting the account balance in the contract configuration.

Reply:
Nice catch. You are correct. I thought you actually changed everything to be fair. I'm thinking that maybe we should finish what you started and handle ALL balances/gas inside config so that we can remove the balance handling in other modules (eg: dev_funds)

Reply:
@jrainville sounds good to me! I think that dev_funds.js conversion is the only one outside of lib/core/config.js now. I'll go ahead and make that change unless anyone has any issues with that 👍 @PascalPrecht